### PR TITLE
Dependency fixes for py3

### PR DESCRIPTION
**Pull request by zacsimile**
In reference to python-microscopy/python-microscopy#259.

### CODE REVIEW

File: pyme-depends/meta.yaml [hunk 1]
```diff
@@ -128,15 +128,18 @@ requirements:
   run:
     - python {{ python }}
     - numpy >=1.11
+    - numpy >= 1.4 [py3k]
```
Comment (David-Baddeley) [inline pyme-depends/meta.yaml]: why do we need a higher numpy on py3?

File: pyme-depends/meta.yaml [hunk 2]
```diff
@@ -128,15 +128,18 @@ requirements:
...
     - python {{ python }}
     - numpy >=1.11
+    - numpy >= 1.4 [py3k]
     - scipy
     - matplotlib
     - wxpython
     #<3.9 [py2k] #FIXME - there are still some issues with display logic on wx=4.x
     #- wxpython-phoenix < 3.9 [py3k]
     #- wxpython [py3k] #FIXME - there are still some issues with display logic on wx=4.x
+    - wxpython <= 4.0.4 [py3k]
```
Comment (David-Baddeley) [inline pyme-depends/meta.yaml]: Is this exclusive to py3? And do we really want to pin?

File: pyme-depends/meta.yaml (hunk 1 above)
Comment (zacsimile) [reply pyme-depends/meta.yaml]: I thought you'd be more receptive to letting us upgrade numpy on py3. Ideally we'd pin to 1.4 or higher on py2 as well.

File: pyme-depends/meta.yaml (hunk 2 above)
Comment (zacsimile) [reply pyme-depends/meta.yaml]: Again, thought you'd be more receptive if we just changed this for py3. We definitely need to pin. Any version higher and nothing works, and it's been that way for months. The default packages always install a newer version of wxpython and the default install is always busted until I downgrade.

Comment (David-Baddeley) [reply pyme-depends/meta.yaml]: Just for the record, could you clarify what you mean by 'nothing works' - is it just traits, or does it effect other stuff too.

Comment (zacsimile) [reply pyme-depends/meta.yaml]: By nothing works I mean I can't launch `pymevisualize`, `dh5view`, or `pymeacquire`. I get an error before the program even comes up.

**Pull request merged**